spec/javascript/genre_test.js

```diff
@@ -0,0 +1,22 @@
+import Genre from '../../app/javascript/components/Genre';
+
+describe('Genre', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <Genre
+        name = "Arts"
+      />
+    );
+  });
+
+//  new test
+  it('should render the Genre component', () => {
+    expect(wrapper.find(Genre)).toBePresent();
+  });
+
+  it('receives the correct props', () => {
+    expect(wrapper.find(Genre).props()).toEqual({name: "Arts"})
+  });
+});
```

Comment:
This seems to be why tests are failing - no component named Genre!
When you rework this, can you test what shows up on page - I.e. a header showing the name, etc?